### ISSUE TITLE
Horizontal scrollbar appears on checkbox/radio buttons with long label

### ISSUE THREAD

**Issue by nickjvm** [grommet/grommet#1338]
### Expected Behavior
Long checkbox/radio labels should wrap without a horizontal scrollbar.

### URL, screen shot, or Codepen exhibiting the issue
![screenshotsfindermay 3 2017 3 19 47 pm](https://cloud.githubusercontent.com/assets/4998403/25682063/f45eb6ec-3013-11e7-95f1-310560cd593b.png)

https://codepen.io/nickjvm/pen/EmvwBW?editors=0110

I believe this is due to the right margin on `.grommetux-check-box__label` (24px) and right margin on `.grommetux-check-box__control` (12px) causing the label to overflow the `FormField` by 36px.

using `calc` to determine the width of the label resolves the issue: 

```css
.grommetux-check-box__label { width: calc(100% - 36px); }
.grommetux-radio-button__label { width: calc(100% - 36px); }
```

### Your Environment

* Grommet version: latest
* Browser Name and version: Chrome Beta
* Operating System and version (desktop or mobile): OS X 10.11.6

**Comment by dgennetten** [comment]
PR #1344 solves this PC-only issue. It should be failing on all browsers!

**Comment by dgennetten** [comment]
@alansouzati This is fixed with #1344. May 16. Let me know if there are questions/issues.
